be/src/storage/task/engine_clone_task.cpp

```diff
@@ -71,9 +71,12 @@ EngineCloneTask::EngineCloneTask(MemTracker* mem_tracker, const TCloneReq& clone
 
 Status EngineCloneTask::execute() {
     MemTracker* prev_tracker = tls_thread_status.set_mem_tracker(_mem_tracker.get());
-    DeferOp op([&] { tls_thread_status.set_mem_tracker(prev_tracker); });
-
     auto tablet_manager = StorageEngine::instance()->tablet_manager();
+    DeferOp op([&] {
```

Comment:
This should be defined after `tablet_manager->register_clone_tablet(_clone_req.tablet_id);`

Reply:
ok, done